assets/js/type-defs/cart.js

```diff
@@ -45,4 +45,38 @@
  * @property {string} email      The email for the billing address
  */
 
+/**
+ * @typedef {Object} CartData
+ *
+ * @property {Array}      coupons       Coupons applied to cart.
+ * @property {Array}      items         Items in the cart.
+ * @property {number}     itemsCount    Number of items in the cart.
+ * @property {number}     itemsWeight   Weight of items in the cart.
+ * @property {boolean}    needsShipping True if the cart needs shipping.
+ * @property {CartTotals} totals        Cart total amounts.
+ */
+
+/**
+ * @typedef {Object} CartTotals
+ *
+ * @property {string} currency_code               Currency code (in ISO format) for returned prices.
+ * @property {string} currency_symbol             Currency symbol for the currency which can be used to format returned prices.
+ * @property {number} currency_minor_unit         Currency minor unit (number of digits after the decimal separator) for returned prices.
+ * @property {string} currency_decimal_separator  Decimal separator for the currency which can be used to format returned prices.
+ * @property {string} currency_thousand_separator Thousand separator for the currency which can be used to format returned prices.
+ * @property {string} currency_prefix             Price prefix for the currency which can be used to format returned prices.
+ * @property {string} currency_suffix             Price prefix for the currency which can be used to format returned prices.
```

Comment:
Nit: these comments go over the line length. Non-blocking though, so up to you if you fix.